setup.py

```diff
@@ -114,7 +114,7 @@ def read_version_py(infname):
     "attrs >= 18.2.0",
 
     # WebSocket library for twisted and asyncio
-    "autobahn >= 19.5.2",
+    "autobahn < 22.4.1",  # remove this when https://github.com/crossbario/autobahn-python/issues/1566 is fixed
```

Comment:
might be worth changing the comment to "when 22.4.3 comes out" since that bug _is_ fixed now, just not released. It will be in >= 22.4.3 though.

Reply:
Will do that in next PR.